src/python/traces/sampling/app.py

```diff
@@ -0,0 +1,31 @@
+from opentelemetry import trace
+from opentelemetry.sdk.resources import Resource
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import BatchSpanProcessor
+from opentelemetry.exporter.otlp.proto.grpc.trace_exporter import OTLPSpanExporter
+
+from opentelemetry.sdk.trace.sampling import TraceIdRatioBased
+
+# sample 1 in every 1000 traces
+sampler = TraceIdRatioBased(1 / 1000)
+# Creates a resource and adds it to the tracer provider
```

Comment:
```suggestion

# Creates a resource and adds it to the tracer provider
```

I'd give spaces between the lines so it's easier to read. Now it looks a bit all cramped together :)

Reply:
Fixed